README.md

```diff
@@ -1,63 +1,57 @@
 # Talos
-A multi level authorization project
+A file system permission project.
 
 ## The concept
-The authorization in this project is an attempt to conveniently define what users can access what resources. Resources are uniquely identified using a "resource identifier", which is a string, possibly consisting of multiple levels separated by slashes in order to keep them well-organized. The users in the systeam each have a role, which also can inherit permissions of another role. The permissions then can be specified using so-called permission rules. For example: 
+This project aims to provide a convenient description mechanism for file system permissions. Resources are identified using a string consisting of one or more levels separated by slashes. Users of the system each have a role which can inherit permissions of other roles. Permissions are specified using so-called permission rules, which are triplets of a boolean (allow or deny), a role and a resource. For example: 
 
 ```
-Admin > User
+Student > Mara
+Student > Jeffrey
 
-allow User  /
-deny  User  /profile
-allow User  /profile/[id]
-allow Admin /profile
-deny  Admin /profile/*/password
-allow Admin /profile/[id]/password
+allow Admin   /
+allow Student /home/[id]
+allow Mara    /srv/nfs/music
+deny  Jeffrey /home/[id]/config
+deny  Admin   /home/*/personalsecrets
```

Comment:
Mara and Jeffrey are just roles for one person, I assume?
This would render the need for using the `[id]` variable for Jeffrey superfluous, since the id of Jeffrey (and therefore the value of the `[id]` variable if Jeffrey is the authenticated user) is already known beforehand.